README.md

```diff
@@ -1,23 +1,33 @@
 # Galaxy plugin for Battle.net
 
+This plugin allows you to install and launch your Blizzard games via the GOG Galaxy 2.0 launcher.
+
 ## Installation
-1. Download [latest](https://github.com/FriendsOfGalaxy/galaxy-integration-battlenet/releases) release of the plugin for your platform.
+1. Download [latest](https://github.com/FriendsOfGalaxy/galaxy-integration-blizzard/releases) release of the plugin for your platform.
 2. Create plugin folder (if it does not exists yet):
 	- Windows: `%LOCALAPPDATA%\GOG.com\Galaxy\plugins\installed\battlenet`
-	- MacOS: `${HOME}/Library/Application Support/GOG.com/Galaxy/plugins/installed/battlenet`
+	- MacOS: `${HOME}/Library/Application Support/GOG.com/Galaxy/plugins/installed/blizzard`
 3. Unpack plugin to the plugin folder created in step 2.
 4. Re-connect(or re-start) your GOG Galaxy Client
 
 ### From source (tested on windows 10 and macos 10.14 with python 3.7)
 - copy / clone this repo
 - run:
 ```bash
-cd balaxy_blizard_plugin
+cd galaxy_blizzard_plugin
 pip install -r requirements/dev.txt
 inv install
 ```
 
-## Developement
+## Note on playtime
+
+Currently we only support displaying the total playtime of your quickplay matches in Overwatch. Playtime might take some time to update after you quit the game.
```

Comment:
Is it one-season only, right? We should state about this.

Reply:
Only competitive is one season (that's why we can't add it to playtime). Quickplay is all-time and does not reset.

Reply:
ah, I see